doc/development.rst

```diff
@@ -276,10 +277,17 @@ These two states are handled in the transition function:
         self.barebox.boot("")
         self.barebox.await_boot()
         self.target.activate(self.shell)
+    elif status == Status.off:
+        if self.status == Status.barebox:
+            self.target.deactivate(self.barebox)
+        elif self.status == Status.shell:
+            self.target.deactivate(self.shell)
+        self.target.activate(self.power)
```

Comment:
You can call deactivate unconditionally, as it's a no-op if the driver isn't active. This applies below as well.